browser/ui/webui/settings/import_feature.cc

```diff
@@ -10,9 +10,8 @@
 
 namespace settings {
 
-BASE_FEATURE(kParallelImports,
-             "ParallelImports",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+const base::Feature kParallelImports{"ParallelImports",
```

Comment:
Looks like this reverted to the pre-cr108 code. It should use BASE_FEATURE.

Reply:
done